### DISCUSSION TITLE
nmap  -oX 提示“flag provided but not defined: -oX”

### DISCUSSION THREAD

**Discussion by Ashro-one** [projectdiscovery/naabu#819] [Q&A]
![image](https://github.com/projectdiscovery/naabu/assets/49979071/22d73eda-fd7e-4b4a-86c2-6af83ed28c16)

**Comment by MetzinAround** [comment]
Looks like it's trying to read the flag as a naabu flag instead of an nmap one. I think it has something to do with the escape slashes OR nmap isn't installed on the machine.
 
Double check that nmap is installed, then if you run the command from the directory you want the file to go in, and only name the file without the slashes, I think it will work. I just tried that. 

![Screenshot -1003 at 14 01](https://github.com/projectdiscovery/naabu/assets/65838556/5dda56ec-3f8e-4c7f-baa5-792d7af75038)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
